recipes/freva-rest-server/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "freva-rest-server" %}
+{% set version = "2411.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/FREVA-CLINT/freva-nextgen/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: 7227d57d637e67c1204e00e9e2aae6f4169a0df1e004c01f0e58172a354f3e9a
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install ./freva-rest -vv --no-deps --no-build-isolation
+
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - setuptools
+    - pip
+
+  run:
+    - python >={{ python_min }}
+    - aiohttp
+    - apache-solr
+    - dask
+    - email-validator
+    - fastapi
+    - fastapi-third-party-auth
+    - mongodb
+    - motor >=3.6
+    - mysql-server
+    - pyjwt
+    - pymongo >=4.9
+    - python-dotenv
+    - python-dateutil
+    - python-multipart
+    - redis-py
+    - redis-server
+    - requests
+    - rich
+    - rich-argparse
+    - setuptools
+    - tomli
+    - typing_extensions
+    - uvicorn
+    - zarr
+
+test:
+  imports:
+    - freva_rest
+  commands:
+    - freva-rest-server --help
+  requires:
+    - python {{ python_min }}
+    - pip
+
+
+about:
+  summary: Server for the Free Evaluation and Analysis Framework (Freva)
+  home: https://github.com/FREVA-CLINT/freva-nextgen
+  license: BSD-2-Clause
```

Comment:
Is this really BSD-2-Clause? Please check upstream: https://github.com/FREVA-CLINT/freva-nextgen/blob/v2411.0.0/LICENSE

Reply:
Of course, good catch!